.github/workflows/ci.yaml

```diff
@@ -80,6 +80,39 @@ jobs:
           version: v2.4.0
           args: --timeout=5m
 
+  lint-jsonnet:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@08eba0b27e820071cde6df949e0beb9ba4906955 # v4
+        with:
+          persist-credentials: false
+      - name: Install jsonnet
+        run: |
+          JSONNET_VERSION=0.21.0
+          curl -sL "https://github.com/google/jsonnet/releases/download/v${JSONNET_VERSION}/jsonnet-v${JSONNET_VERSION}.tar.gz" | tar -xz
+          cd jsonnet-v${JSONNET_VERSION}
+          make > /dev/null
```

Comment:
We could also use `go-jsonnet` and its precompiled binaries here to save some time.